first version blog post (#555)
kestra-io/docs: content/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.md

---

> title: "Kestra Raises $3 Million to Build the Universal Open Source Orchestration Solution for All Engineers"

Reviewer: ```suggestion
title: "Kestra Raises $3 Million to Simplify and Unify Orchestration for All Engineers"
```

to build makes it sound as we start to build, the product already exists so maybe this

> description: "Kestra raises $3 million in funding to build a universal open-source orchestration platform, empowering software and data engineers across industries."

Reviewer: ```suggestion
description: "Kestra raises $3 million in funding to continue building a language-agnostic, universal open-source orchestration platform, empowering software and data engineers across industries."
```

date: 2023-10-05T10:00:00
category: News & Products Updates
author:
  name: Emmanuel Darras
  image: "edarras"
image: /blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.jpg
---

> We are thrilled to announce that Kestra has raised **$3 million of funding** to build the leading universal open-source orchestrator. We aim to empower a wide range of users, from software developers to data engineers, by simplifying and unifying their automation and orchestration needs. We envision Kestra as the go-to solution for organizations seeking **simple** and **powerful** orchestration solutions for any kind of process and data automation usage.

Reviewer: ```suggestion
We are thrilled to announce that Kestra has raised **$3 million of funding** to make orchestration more accessible, collaborative, and scalable. 

We aim to empower a wide range of users, from software developers to data engineers, by simplifying and unifying their automation and orchestration needs. We envision Kestra as the go-to solution for organizations seeking **simple** and **versatile** orchestration.

Through a seamless UI and IDE access, all workflows — from those orchestrating mission-critical applications, business processes, and data pipelines to simple automations — can be version controlled and governed using engineering best practices.
```

not strong enough as intro yet -- maybe this to show the problem kestra tries to solve?

Author: Yep, I stick to the PR for the content as every word was chosen, but I like also being straight to the point and informative

> ## Trusted by Companies of All Sizes

Reviewer: ```suggestion
## Trusted by Enterprises
```

Since its inception, Kestra has experienced significant traction as an open-source project, thriving with a vibrant community of users and contributors. Thousands of companies, from startups to large enterprises, including **Huawei**, **Leroy Merlin**, **Acxiom**, **Tencent**, **Gorgias**, **Sophia Genetics**, and **Decathlon**, are already orchestrating their **mission-critical workloads** with Kestra.

**Images avec une liste des clients**

> ## Breaking Down the Barriers of hyperspecialization

Reviewer: ```suggestion
## Breaking Down the Orchestration Silos
```

## Fast, Reliable, and Extensible

Kestra strives to be the easiest-to-use orchestration tool on the market. Our users construct their first scheduled or event-driven workflow in a matter of minutes via our intuitive user interface or API, thereby obviating the need for complex deployment processes. We are steadfast in our belief that a simple orchestration API leads to greater reliability and extensibility than convoluted client-side frameworks.

## What's Coming Next

> The introduction of a SaaS offering is among our upcoming milestones, further simplifying the user experience by removing initial setup complexities. This will serve as an additional option alongside our existing solutions, catering to a wider range of user needs.

Reviewer: ```suggestion
The introduction of a fully-managed offering is among our upcoming milestones, further simplifying the user experience by removing the complexity of managing infrastructure. This will serve as an additional option alongside our existing [Kestra Enterprise](url) solution.
```

We're also planning to expand and refine features that facilitate intra-organizational collaboration and improve operational efficiency.

We're incredibly grateful for the support and confidence shown by our investors, community, and customers. This funding enables us to further our vision of making Kestra the go-to orchestration platform.

If you want to participate to Kestra success, give us a [GitHub star](https://github.com/kestra-io/kestra) and join [the community](https://kestra.io/slack).

## Trusted by Leading Tech Investors

This funding was co-led by [ISAI](http://www.isai.vc/), [Axeleo Capital](https://www.axc.vc/), alongside several SaaS and Open source specialists angel investors, including founders and executives from Datadog (Olivier Pomel, Co-Founder and CEO), Collibra (Stan Christians, Co-founder & Chief Data Citizen), Strapi (Pierre Burgy, Co-Founder and CEO), BlaBlaCar (Olivier Bonnet, CTO) and many more.

> We are honored to have the support and trust of an exceptional group of investors, deeply rooted in the technology sector. Below is a list of these esteemed individuals who believe in Kestra's mission and potential.

Reviewer: ```suggestion
We are honored to have the support and trust of an exceptional group of investors, deeply rooted in the technology sector, including:
```

description: "Kestra raises $3 million in funding to continue building a language-agnostic, universal open-source orchestration platform, empowering software and data engineers across industries."
date: 2023-10-05T10:00:00
category: News & Products Updates
author:
  name: Emmanuel Darras
  image: "edarras"
image: /blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.jpg
---

We are thrilled to announce that Kestra has raised **$3 million of funding** to make orchestration more accessible, collaborative, and scalable. 

We aim to empower a wide range of users, from software engineers to data professionals, by simplifying and unifying their automation and orchestration needs. We envision Kestra as the go-to solution for organizations seeking **simple** and **versatile** orchestration.

Through a seamless UI and IDE access, all workflows — from those orchestrating mission-critical applications, business processes, and data pipelines to simple automations — can be version controlled and governed using engineering best practices.

## Trusted by Companies of All Sizes

> Since its inception, Kestra has experienced significant traction as an open-source project, thriving with a vibrant community of users and contributors. Thousands of companies, from startups to large enterprises, including **Huawei**, **Leroy Merlin**, **Acxiom**, **Tencent**, **Gorgias**, **Sophia Genetics**, and **Decathlon**, are already orchestrating their **mission-critical workloads** with Kestra.

Reviewer: ```suggestion
Since its inception, Kestra has experienced significant traction as an open-source project, thriving with a vibrant community of users and contributors. Thousands of companies, from startups to large enterprises, including **Huawei**, **Leroy Merlin**, **Acxiom**, **Tencent**, **Gorgias**, **Sophia Genetics**, and **Decathlon**, are already orchestrating their **business-critical workloads** with Kestra.
```

![clients](/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator/clients.png)

## Breaking Down the Orchestration Silos

[Kestra](https://github.com/kestra-io/kestra)’s innovative approach to orchestration aims to overcome the challenges posed by hyperspecialization in various domains — be it **data orchestration**, **process** and **microservice** orchestration, or **infrastructure** and **business automation**. Such hyperspecialization has often resulted in organizational silos, hindering collaboration and operational excellence.

To address this, Kestra offers a versatile and language-agnostic developer platform while simultaneously providing an intuitive user interface tailored to business professionals. This holistic solution aims to bridge the gap between different organizational units, thereby fostering a collaborative environment conducive to greater operational efficiency.

> ## The Orchestration Platform for Mission-Critical Operations

Reviewer: ```suggestion
## The Orchestration Platform for Business-Critical Operations
```

![Kestra Baseline](/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator/baseline.png)

> Our initial mission was to build the most efficient open-source workflow platform targeting the data engineering space. Since then, we’ve experienced tremendous growth well beyond that domain. Our users leverage Kestra not only to orchestrate data pipelines but also to power their **mission-critical applications**. They use our platform to orchestrate microservices and applications feeding data to their end products, and then they use Kestra for analytical workflows to understand how their products are performing. This way, Kestra serves as a **universal orchestration** solution for their business.

Reviewer: ```suggestion
Our initial mission was to build the most efficient open-source workflow platform targeting the data engineering space. Since then, we’ve experienced tremendous growth well beyond that domain. Our users leverage Kestra not only to orchestrate data pipelines but also to power their **business-critical applications**. They use our platform to orchestrate microservices and applications feeding data to their end products, and then they use Kestra for analytical workflows to understand how their products are performing. This way, Kestra serves as a **universal orchestration** solution for their business.
```
